Update to Cake 5.0

twenzel opened the pull request:
Fixes #152

twenzel committed:
Update to Cake 5.0

twenzel committed:
Update GH action

twenzel committed:
Install .NET 9 SDK

twenzel commented:
@augustoproiete Unfortunately the Cake.Recipe stuff does not work with Cake 5.0. Any suggestions?

gep13 commented:
@twenzel hopefully you don't have any objections, but I am going to take this PR for a spin, to try to get this over the line.

While the Cake.FileHelpers addin is looking to target Cake v5.0.0, there is no requirement for changing the version of Cake that is doing the actual build.  Cake.Recipe naturally run slightly behind the version of Cake.

gep13 committed:
(#153) Update to latest Cake.Recipe

Version 4.0.0, which is targeting Cake v2.x, so pin to 2.3.0.

gep13 committed:
(maint) Update NuGet packages

gep13 committed:
(build) Update build images

The previous ones no longer exist.

gep13 committed:
(build) Overhaul workflow files

Making them the same as what is run in Cake.AppVeyor.

gep13 committed:
(test) Add basic testing script

Just to make sure that the addin is being loaded, and is capable of
executing at least one of the aliases provided by the addin.

gep13 reviewed (dismissed):
LGTM!

gep13 committed:
(build) Remove dependabot

Switching to Renovate instead.

gep13 reviewed (approved):
LGTM!
